swap_meet/vendor.py

```diff
@@ -1,2 +1,66 @@
 class Vendor:
-    pass
+    def __init__(self, inventory=None):
+
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+
+    def get_by_category(self, category):
+        category_list = []
+        for item in self.inventory:
+            if category == item.category:
+                category_list.append(item)
+
+        return category_list
+
+    def swap_items(self, someonelse, my_item, someonelse_item):
+        if my_item in self.inventory and someonelse_item in someonelse.inventory:
```

Comment:
Consider reversing the sense of this comparison and using it as a guard clause like you did in `remove`.

In general, consider reversing the sense of any logic like this where the else is a simple return so that the return is part of the if. Since it returns when run, we don't need to use the else keyword, and we can avoid indenting our main function logic.

This is a very common pattern in production code.